src/tests/e2e/users.get.test.ts

```diff
@@ -0,0 +1,31 @@
+import request from 'supertest';
+import app from '../../app';
+import { mockValidToken, generateToken } from '../mockObjects/mockToken';
+import { StatusCodes } from 'http-status-codes';
+import { setupE2E } from '../setupTests';
+import { userService } from '../../services/userService';
+import { mockGetAllUsers } from '../mockObjects/mockUserService';
+
+setupE2E('usersget-e2e', () => {
+  const getAllUsersSpy = jest.spyOn(userService, 'getAllUsers').mockImplementation(mockGetAllUsers);
+  describe('GET /users should', () => {
+    test(`respond with ${StatusCodes.INTERNAL_SERVER_ERROR} when error is thrown`, async () => {
+      getAllUsersSpy.mockRejectedValueOnce('error');
+      const res = await request(app).get('/users').set('Authorization', `Bearer ${mockValidToken}`);
```

Comment:
transfer endpoint path to constant

Reply:
And transfer authorization token `Bearer ${mockValidToken}` to const. That's for all tests